apps/web/pages/auth/setup/steps/ChooseLicense.tsx

```diff
@@ -0,0 +1,76 @@
+import { DeploymentLicenseType } from "@prisma/client";
+import classNames from "classnames";
+import { useRouter } from "next/router";
+import { Dispatch, SetStateAction } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+
+const ChooseLicense = (props: {
+  isFreeLicense: boolean;
+  setIsLoading: Dispatch<SetStateAction<boolean>>;
+  setIsFreeLicense: Dispatch<SetStateAction<boolean>>;
+}) => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const { isFreeLicense, setIsFreeLicense } = props;
+  const mutation = trpc.viewer.auth.deploymentSetup.useMutation({
+    onSuccess: () => {
+      router.replace(`/auth/setup?step=3${isFreeLicense ? "&category=calendar" : ""}`);
+    },
+  });
+
+  const enterprise_fee = "$99"; // TODO: get this from a new API endpoint
```

Comment:
@leog @zomars this needs to come from our backend so we can update them dynamically